e2e/tests/conceptsets/conceptset-edit.spec.ts

```diff
@@ -19,13 +19,13 @@ describe('Editing and Copying Concept Sets', () => {
 
   /**
    * Test:
-   * - Create new workspace.
+   * - Find an existing workspace.
    * - Create first new Concept Set in Procedure domain.
    * - Create second new Concept Set in Procedure domain. Add to first Concept Set.
    */
   test('Add to existing Concept Set in same workspace', async () => {
     // Create a workspace
-    const workspaceName = await findWorkspace(page, {create: true}).then(card => card.clickWorkspaceName());
+    const workspaceName = await findWorkspace(page).then(card => card.clickWorkspaceName());
```

Comment:
This doesn't seem related to the ellipsis -> snowman renaming.

Reply:
you're right. I can revert the change. I thought this test doesn't need a new workspace to run.

Reply:
Yes, please split this into a separate PR

Reply:
Done.